packages/enzyme-test-suite/test/_helpers/htmlMap.js

```diff
@@ -0,0 +1,100 @@
+/* eslint-env browser */
+const map = {
+  a: HTMLAnchorElement,
+  audio: HTMLAudioElement,
+  area: HTMLAreaElement,
+  br: HTMLBRElement,
+  base: HTMLBaseElement,
+  basefont: HTMLBaseFontElement,
+  body: HTMLBodyElement,
+  button: HTMLButtonElement,
+  canvas: HTMLCanvasElement,
+  // content: HTMLContentElement,
+  dl: HTMLDListElement,
+  data: HTMLDataElement,
+  datalist: HTMLDataListElement,
+  dialog: HTMLDialogElement,
+  div: HTMLDivElement,
+  embed: HTMLEmbedElement,
+  fieldset: HTMLFieldSetElement,
+  // 'formcontrol': HTMLFormControlsCollection,
+  form: HTMLFormElement,
+  frameset: HTMLFrameSetElement,
+  hr: HTMLHRElement,
+  head: HTMLHeadElement,
+  h1: HTMLHeadingElement,
+  h2: HTMLHeadingElement,
+  h3: HTMLHeadingElement,
+  h4: HTMLHeadingElement,
+  h5: HTMLHeadingElement,
+  h6: HTMLHeadingElement,
+  html: HTMLHtmlElement,
+  iframe: HTMLIFrameElement,
+  img: HTMLImageElement,
+  input: HTMLInputElement,
+  // isindex: HTMLIsIndexElement,
+  // keygen: HTMLKeygenElement,
+  li: HTMLLIElement,
+  label: HTMLLabelElement,
+  legend: HTMLLegendElement,
+  link: HTMLLinkElement,
+  map: HTMLMapElement,
+  media: HTMLMediaElement,
+  meta: HTMLMetaElement,
+  meter: HTMLMeterElement,
+  del: HTMLModElement,
+  ins: HTMLModElement,
+  ol: HTMLOListElement,
+  object: HTMLObjectElement,
+  optgroup: HTMLOptGroupElement,
+  option: HTMLOptionElement,
+  // 'options': HTMLOptionsCollection,
+  output: HTMLOutputElement,
+  p: HTMLParagraphElement,
+  param: HTMLParamElement,
+  // picture: HTMLPictureElement,
+  pre: HTMLPreElement,
+  progress: HTMLProgressElement,
+  blockquote: HTMLQuoteElement,
+  q: HTMLQuoteElement,
+  cite: HTMLQuoteElement,
+  script: HTMLScriptElement,
+  noscript: HTMLScriptElement,
+  select: HTMLSelectElement,
+  // shadow: HTMLShadowElement,
+  source: HTMLSourceElement,
+  span: HTMLSpanElement,
+  style: HTMLStyleElement,
+  caption: HTMLTableCaptionElement,
+  th: HTMLTableCellElement,
+  td: HTMLTableCellElement,
+  col: HTMLTableColElement,
+  colgroup: HTMLTableColElement,
+  table: HTMLTableElement,
+  tr: HTMLTableRowElement,
+  tfoot: HTMLTableSectionElement,
+  thead: HTMLTableSectionElement,
+  tbody: HTMLTableSectionElement,
+  template: HTMLTemplateElement,
+  textarea: HTMLTextAreaElement,
+  time: HTMLTimeElement,
+  title: HTMLTitleElement,
+  track: HTMLTrackElement,
+  ul: HTMLUListElement,
+  bgsound: HTMLUnknownElement,
+  blink: HTMLUnknownElement,
+  multicol: HTMLUnknownElement,
+  nextid: HTMLUnknownElement,
+  rb: HTMLUnknownElement,
+  spacer: HTMLUnknownElement,
+  video: HTMLVideoElement,
+};
+
+/**
+ * returns the map
+ */
+export function getMap() {
```

Comment:
this should be the default; and the object literal should be created inside the `getMap` function.